fundamentals/fcsreqs.tex

```diff
@@ -1,4 +1,52 @@
 
+\subsubsection{Open Access and Development Practices}
+
+The proprietary concerns of research institutions and security constraints of
+data within fuel cycle simulators often restrict access. Use of a simulator is
+therefore often limited to its institution of origin, necessitating effort
+duplication at other institutions and thereby squandering broader human
+resources. License agreements and institutional approval are required for most
+current simulators (i.e. \gls{COSI}6, \gls{DANESS}, \gls{DESAE}, EVOLCODE,
+FAMILY21, \gls{NFCSim})\cite{juchau_modeling_2010}, ORION, and VISION.  Even when, as in
+the case of the MIT \gls{CAFCA} software, the source code is unrestricted, the
```

Comment:
Maybe moving VISION alongside CAFCA would help with that concern about whether or not vision still needs institutional approval (though, I do think you have to ask Idaho for VISION still)